src/hippopt/base/optimization_object.py

```diff
@@ -0,0 +1,64 @@
+from hippopt.common import (
+    Any,
+    ClassVar,
+    Type,
+    TypeVar,
+    Union,
+    abc,
+    copy,
+    cs,
+    dataclasses,
+    np,
+)
```

Comment:
The reason why I wanted to do something like that is to have consistency across the entire project. In this way, I can ensure that the same alias (e.g. ``cs``) is used across the project. in this way, if by any chance, we wanted to change an import, we can edit in a single place. 

Is there any other alternative to make sure to have consistency across the project while avoiding to copy paste stuff

Reply:
What do you mean with _consistency_? Even if you do so, there would be nothing preventing contributing users to import packages directly, and I already see countless comments from your side to basically all PRs since it's very uncommon to import cs/np (and especially typing/default modules) not directly, and users will keep doing it.

Another reason why you might not want to do it is that is a good practice reduce the inter-module dependency as much as possible. I would not wish to my worse enemy to get a circular import error on a medium-size Python project.

In general, the usage of `__all__` is useful when you want to differentiate public from private modules, that at this stage is really premature (see [`jax`](https://github.com/google/jax/blob/main/jax/__init__.py) and `jax._src` as a good example for this).

Reply:
Ok fair enough! I suspect I was not predicting a series of side effects. I removed ``common.py`` in https://github.com/ami-iit/hippopt/pull/3/commits/09ecc8e6096016e0c62b6b8dea9822e613e27ebc